### PR TITLE
Allow list dialog items.size() = 0 rather than not initializing builder.items

### PR DESCRIPTION
**Pull request by smartjinyu**
Sometimes we need to create a list dialog (SingleChoice/ MultiChoice) with no default items in it. Like "BASIC LIST (LONG PRESS)" in sample app, we can allow users to add items manually. If we write the following code
`List<String> list = new ArrayList<>();
MaterialDialog.Builder(this).items(list).show();`
then builder.items will not initialized ,and we will get a null object if we call dialog.getItems() method later.
In this pr, I try to initialize builder.items as a List with no elements, and dialog.getItems() will not return null then.

### CODE REVIEW

File: core/src/main/java/com/afollestad/materialdialogs/DialogInit.java [hunk 1]
```diff
@@ -57,7 +57,7 @@ static int getTheme(@NonNull MaterialDialog.Builder builder) {
     static int getInflateLayout(MaterialDialog.Builder builder) {
         if (builder.customView != null) {
             return R.layout.md_dialog_custom;
-        } else if (builder.items != null && builder.items.size() > 0 || builder.adapter != null) {
+        } else if (builder.items != null && builder.items.size() >= 0 || builder.adapter != null) {
```
Comment (afollestad) [inline core/src/main/java/com/afollestad/materialdialogs/DialogInit.java]: `builder.items.size() >= 0` isn't necessary, there's never a case where the length of an array will be *less than* 0. That check can be taken out.

File: core/src/main/java/com/afollestad/materialdialogs/MaterialDialog.java [hunk 2]
```diff
@@ -766,6 +766,8 @@ public Builder items(@NonNull Collection collection) {
                     i++;
                 }
                 items(array);
+            }else if(collection.size()==0){
```
Comment (afollestad) [inline core/src/main/java/com/afollestad/materialdialogs/MaterialDialog.java]: Standard formatting needs to be used here. 

```java
} else i f(collection.size() == 0 ) {
```

Android Studio's Reformat Code function will do that for you.

**Comment by smartjinyu** [comment]
Sorry for my carelessness. Thanks for your advice. And I will pay attention to that in my future development.

**Comment by afollestad** [comment]
No worries, thanks for the contribution!

**Pull request merged**